server/GRPC_TA2_TA3/relay.js

```diff
@@ -0,0 +1,58 @@
+// Javascript Imports
+const path = require("path");
+const grpc = require("grpc");
+const WebSocketServer = require("ws").Server;
+
+// Environment variables for GRPC
+let GRPC_PORT = process.env.GRPC_PORT
```

Comment:
These three lines does not have colons at EOL. Besides since they are constants, "const" is preferred over "let".